### PR TITLE
Buttonify

### PR DESCRIPTION
**Pull request by yte0**
Add filtering by achievement level

### CODE REVIEW

File: R/mod_quality_overview.R [hunk 1]
```diff
@@ -270,4 +286,32 @@ mod_quality_overview_server <- function(input,
...
+
+    level_buttons <- lapply(
+      names(clicked_level),
+      function(button) {
+        shiny::observeEvent(
+          shiny::req(input[[button]]), {
+            if (clicked_level[[button]]) {
+              clicked_level <<- list(F, F, F)
+              names(clicked_level) <<- c("legend_high", "legend_mod", "legend_low")
+              filter_indicator$level <- list("H", "M", "L", "undefined")
```
Comment (arnfinn) [inline R/mod_quality_overview.R]: These lines are not tested. Is it possible to add test?

Comment (yte0) [reply R/mod_quality_overview.R]: I dont know how to do it. That part is activated when the buttons are clicked for the second time.

**Comment by yte0** [comment]
version 0.4.1?

**Comment by arnfinn** [comment]
It is a feature, so it will be `0.5.0`:) You put it under `Unreleased` and then we change it directly on the `master` branch when we create a new branch. Maybe under a subsection `New features` like this?:
```
# Unreleased

## New features

* Enable filtering by the achievment level of the qi([#89](https://github.com/SKDE-Felles/qmongr/pull/89))
```

File: R/mod_quality_overview.R (hunk 1 above)
Comment (arnfinn) [reply R/mod_quality_overview.R]: It seems to work now with f869f5b. With this:
```R
session$setInputs(legend_high = 1)
session$setInputs(legend_high = 1) # twice
session$setInputs(legend_mod = 0)
session$setInputs(legend_low = 0)
expect_equal_to_reference(output$qi_table, "data/output_qi_table_filter_level_high_2017.rds")
```

**Pull request merged**